manifests/fpm.pp

```diff
@@ -44,12 +44,12 @@
 #   Defaults is empty hash.
 #
 class php::fpm (
-  $ensure               = $::php::ensure,
+  $ensure               = $::php::params::ensure,
   $service_ensure       = $::php::params::fpm_service_ensure,
   $service_enable       = $::php::params::fpm_service_enable,
   $service_name         = $::php::params::fpm_service_name,
   $service_provider     = undef,
```

Comment:
It'd be nice if there was some consistency in this class.  Why does `service_provider` default to `undef` whilst `service_name` defaults to what's in `params.pp`?